monkey/monkey_island/cc/ui/src/components/report-components/zerotrust/VennDiagram/Tooltip/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+class Tooltip extends React.Component{
+
+    render() {
+        
```

Comment:
It would look much better if we used React Bootstrap Popovers from here: https://react-bootstrap.netlify.com/components/overlays/#popovers, where the title is the Pillar and the content are the numbers. The color should remain based on the status (passed via bcolor like you did).